src/app/rundown/components/rundown/rundown.component.html

```diff
@@ -8,17 +8,12 @@
       <button *ngIf="rundown?.isActive" (click)="deactivateRundown()">Deactivate Rundown</button>
```

Comment:
Would be nice if you changed this button grouping to:
```
<button *ngIf="!rundown.isActive; else activeRundownButtons" (click)="activeRundown()">...</button>
<ng-template #activeRundownButtons>
  <button> (click)="takeNext()">...</button>
  ...
</ng-template>
```

Reply:
Updated.